hazelcast-integration/eureka/springboot-embedded/hazelcast-metadata/pom.xml

```diff
@@ -45,7 +46,7 @@
                 <artifactId>spring-boot-starter-parent</artifactId>
                 <type>pom</type>
                 <scope>import</scope>
-                <version>2.0.6.RELEASE</version>
+                <version>${spring-boot.version}</version>
```

Comment:
This code sample seemed to start fine but threw some errors since it couldn't reach some server so it needs to be verified.
Same as `hazelcast-only` and `hazelcast-separate-client`.